worlds/openrct2/Options.py

```diff
@@ -0,0 +1,862 @@
+    build_your_own_six_flags_park = 141
+    build_your_own_six_flags_over_texas = 142
+    random_RCT1 = 143
+    random_loopy_landscapes = 144
+    random_corkscrew_follies = 145
+    random_RCT2 = 146
+    random_wacky_worlds = 147
+    random_time_twister = 148
+    random_RCT1_expansions = 149
+    random_RCT2_expansions = 150
+    archipelago_madness_vanilla = 151
+    archipelago_madness_expansions = 152
+    
+class LocationBalancingMode(IntEnum):
+    disabled = 0
+    compromise = 1
+    full = 2
+
+class DeathLinkMode(IntEnum):
+    disabled = 0
+    enabled = 1
+
+class Visibility(IntEnum):
+    nothing = 0
+    recipient = 1
+    full = 2
+
+class Difficulty(IntEnum):
+    very_easy = 0
+    easy = 1
+    medium = 2
+    hard = 3
+    extreme = 4
+
+class Preferred_Intensity(IntEnum):
+    less_intense = 0
+    normal = 1
+    more_intense = 2
+
+class Randomization_Range(IntEnum):
+    none = 0
+    low = 1
+    medium = 2
+    high = 3
+    extreme = 4
+
+class Scenario_Length(IntEnum):
+    synchronous_short = 0
+    synchronous_long = 1
+    lengthy = 2
+    marathon = 3
+    
+class Stat_ReRolls(IntEnum):
+    never = 0
+    infrequent = 1
+    semi_frequent = 2
+    frequent = 3
+    very_frequent = 4
+    extremely_frequent = 5
+
+class Difficult_Guest_Generation(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Difficult_Park_Rating(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_High_Construction(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_Landscape_Changes(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_Marketing_Campaigns(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_Tree_Removal(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class OpenRCT2OnToggle(DefaultOnToggle):
+    @property
+    def result(self) -> bool:
+        return self.value
+
+
+class OpenRCT2Toggle(Toggle):
+    @property
+    def result(self) -> bool:
+        return self.value
+
+
+class LocationBalancing(Choice):
+    """Location balancing affects the density of progression items found in your world relative to other worlds. This setting changes nothing for solo games.
+
+    - Disabled: Location density in your world can fluctuate greatly depending on the settings of other players. In extreme cases, your world may be entirely populated with filler items
+
+    - Compromise: Locations are balanced to a midpoint between "fair" and "natural"
+
+    - Full: Locations are balanced in an attempt to make the number of progression items sent out and received equal over the entire game"""
+    auto_display_name = True
+    display_name = "Location Balancing"
+    option_disabled = LocationBalancingMode.disabled.value
+    option_compromise = LocationBalancingMode.compromise.value
+    option_full = LocationBalancingMode.full.value
+    default = LocationBalancingMode.compromise.value
+
+class SelectedScenario(Choice):
```

Comment:
You should probably check how this looks on webhost. I'm hoping it looks fine

Reply:
I ran through it pretty recently. I think everything looks good!